src/mahalanobis.jl

```diff
@@ -32,21 +32,6 @@ end
 
 sqmahalanobis(a::AbstractVector, b::AbstractVector, Q::AbstractMatrix) = SqMahalanobis(Q)(a, b)
 
-function colwise!(r::AbstractArray, dist::SqMahalanobis, a::AbstractMatrix, b::AbstractMatrix)
-    Q = dist.qmat
-    m, n = get_colwise_dims(size(Q, 1), r, a, b)
-    z = a - b
-    dot_percol!(r, Q * z, z)
-end
-
-function colwise!(r::AbstractArray, dist::SqMahalanobis, a::AbstractVector, b::AbstractMatrix)
-    Q = dist.qmat
-    m, n = get_colwise_dims(size(Q, 1), r, a, b)
-    z = a .- b
-    Qz = Q * z
-    dot_percol!(r, Q * z, z)
-end
-
```

Comment:
Why do you say these are "useless"? Shouldn't we make them specializations of `broadcast!` (and `map!` probably too) once JuliaLang/julia#32310 is merged?

Reply:
Yes, sorry, I was too quick with those ones. The Mahalanobis methods are the only useful ones, and, as you say, are another use case for the iterator type. But without lower bounding the package to a Julia version that includes that PR, we can't deprecate colwise for the exact same reason as with `pairwise`.